app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -1,20 +1,20 @@
 module IrsAttemptsApi
   class AttemptEvent
-    attr_accessor :jti, :iat, :event_type, :session_id, :occurred_at, :event_metadata
+    attr_accessor :jti, :iat, :event_type, :session_id, :occurred_at_ms, :event_metadata
 
     def initialize(
       event_type:,
       session_id:,
-      occurred_at:,
+      occurred_at_ms:,
       event_metadata:,
       jti: SecureRandom.uuid,
-      iat: Time.zone.now.to_i
+      iat: Time.zone.now.to_f
```

Comment:
1. I think we should lave the `iat` claim, the standard SET thing alone as whole integer seconds
2. `to_f` is going to be integer seconds plus a few decimals of milliseconds, is that what we want here?

Reply:
@zachmargolis Not to nitpick a typo, but what word is 'lave' supposed to be in your #1? I'm unsure if you mean 'leave' or 'have'.

Reply:
that was supposed to be "leave", whoops

Reply:
`iat` changes have been reverted.